x-pack/plugins/reporting/server/usage/get_reporting_usage.ts

```diff
@@ -161,7 +175,7 @@ export async function getReportingUsage(
                 jobTypes: {
                   terms: { field: JOB_TYPES_FIELD, size: DEFAULT_TERMS_SIZE },
                   aggs: {
-                    appNames: { terms: { field: OBJECT_TYPES_FIELD, size: DEFAULT_TERMS_SIZE } }, // NOTE Discover/CSV export is missing the 'meta.objectType' field, so Discover/CSV results are missing for this agg
```

Comment:
This comment is no longer valid as `meta.objectType` is added for all export types in the RequestHandler class.